.nvmrc

```diff
@@ -0,0 +1,2 @@
+v14.16.0
+engine-strict=true
```

Comment:
I don't know .nvmrc very much. But does this mean that the sdk-base package only works on Node.js v14.16.0 (or later?)? If any pre-requisites exist to use this package, it should be noted in the README (including other SDK packages).

Reply:
@scalarindetail @Torch3333 Can you clarify?

Reply:
My point when I made this change was to set the node/npm version that is used for development purpose. In a way,I would like to communicate to the SDK users that this package has been developed and tested using node 14.16.0, if the user were to use a different node/npm version, we can't guarantee there is no bug as we haven't tested it.

Though, I did not consider the impact it had on the deployed package and its usage as a npm dependency. By setting the version like is now on this PR, when the sdk-base is used as a npm dependency, it will require     
```
"node": ">=14.0.0",
"npm": ">=6.0.0"
```
which is very constraining of course so now I am not sure it is a good idea anymore to set the version as it now.

```
But does this mean that the sdk-base package only works on Node.js v14.16.0 (or later?
```
I think it would work on more recent and somewhat older node version but it's something I can't guarantee as it means we would need to test the whole SDK using such version.

Reply:
Base on the oral discussion I had with @feeblefakie earlier, let me : 
-  update the `.nvmrc` to only contains 
```
v14.16.0
```
- remove the version requirement from `package.json` 
- update the readme.md to mention we are using node v14.16.0 for development and testing so we cannot guarantee the behavior for other versions.

Reply:
@Torch3333 Thank you for clarifying. I agree the modest recommendation/requirement is good as you wrote.